README.zh-TW.md

```
@@ -0,0 +1,355 @@
+# Context window
+
+根據[這篇文章](<https://help.openai.com/en/articles/11909943-gpt-53-and-gpt-54-in-chatgpt#:~:text=Thinking%20(GPT%E2%80%915.4%20Thinking)>)以及[這段程式碼](https://github.com/openai/codex/blob/main/codex-rs/models-manager/src/model_info.rs#L85)，ChatGPT Web 與 Codex 的 context window 並不相同。
+
+| 方案 | CatDesk + ChatGPT Web（in + out = 總和） | Codex CLI（總和）      |
+| ---- | ---------------------------------------- | ---------------------- |
+| Plus | 128K + 128K = 256K                       | 258K（1M experimental） |
+| Pro  | 272K + 128K = 400K                       | 258K（1M experimental） |
+
+# FAQ
+
+### 可以關掉紅色 CSP 按鈕嗎？
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)